# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
# Commit: 155e82611fed
# Full Hash: 155e82611fed1d8d9f67dcc0e492907041142f48
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2021-02-26 09:41:23
# Description:
#   Bug 1694822 - Fix return of RenderCompositorD3D11SWGL::DoCreateTile() r=mattwoodrow
#   
#   "return" was removed by mistake.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106360
# ==============================================================================

diff -r fd69f661c40d -r 155e82611fed gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
--- a/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Fri Feb 26 02:18:32 2021 +0000
+++ b/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Fri Feb 26 03:56:40 2021 +0000
@@ -401,7 +401,7 @@
   MOZ_ASSERT(SUCCEEDED(hr));
   if (!texture) {
     gfxCriticalNote << "Failed to allocate Texture2D: " << aSurface->TileSize();
-    MakeUnique<TileD3D11>(nullptr, nullptr, nullptr, aSurface, this);
+    return MakeUnique<TileD3D11>(nullptr, nullptr, nullptr, aSurface, this);
   }
 
   RefPtr<DataTextureSourceD3D11> source = new DataTextureSourceD3D11(
